march_hardware_interface/config/testsetup/controllers.yaml

```diff
@@ -6,6 +6,9 @@ march:
     temperature_sensor:
       type: march_temperature_sensor_controller/MarchTemperatureSensorController
       publish_rate: 50
+    pdb_state:
+      type: march_pdb_state_controller/MarchPdbStateController
+      publish_rate: 50
```

Comment:
Do we want to launch this for the testsetup?

Reply:
Depending on which testsetup launch file you pick pdb can be launched